src/main/resources/com/tresys/nitf/xsd/nitf.dfdl.xsd

```diff
@@ -45,17 +45,23 @@ The message root is 'NITF'.
 
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
            xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+           xmlns:dfdlx="http://www.ogf.org/dfdl/dfdl-1.0/extensions"
            xmlns:fn="http://www.w3.org/2005/xpath-functions"
            xmlns:daf="urn:ogf:dfdl:2013:imp:daffodil.apache.org:2018:ext"
            xmlns:nitf="urn:nitf:2.1"
+           xmlns:ude="urn:nitfUserDefinedExtensions"
```

Comment:
Thoughts on a prefix like "nitfx" or something? Makes it more clear that this is nitf related? Or maybe even something different? To me, the word "extensions" almost implies an it's optional, but anyone using NITF must create the types for the schema to work. Not sure what a better name would be.